### ISSUE TITLE
Install does not work in Unity

### ISSUE THREAD

**Issue by laultman** [atteneder/glTFast#648]
I have not used Unity in more than two years, so I am practically starting over. I am configuring a new project using the MRTK3 and the online instructions for MRTK3. There are two errors reported when using OpenXR with the project validation both pertaining to the gltFast (... you need KTX support for your glTF importer...). The Edit button links to this repository.

Following the instructions in the project's ReadMe document, opening the unitypackage does not seem to do anything. 
Per the instructions, I downloaded the file. I clicked on the downloaded file with Unity editor open (version 2022.3.9f1) and with the project closed. Neither seemed to do anything.

I can open the Package Manager and there is no gltfast listing in any of the selections. There is no My Registries option in any of the Unity Package options.

Evidently the documentation is in error or out of date. In any case, I am unable to add it to a project, the instructions are insufficient for me to figure out how to add the component.

**To Reproduce**
1. Create a new project in editor 2022.3.9f1
2. Use the Microsoft instructions for MRTK3 to create the project configuration.
3. Validate the project configuration.

**Comment by atteneder** [comment]
Hi @laultman ,

Opening the installation unitypackage should give you a dialog about adding a bunch of files (e.g. scripts). After adding them (all of them!), the script executes, adds a scoped package registry and finally the package.
Can you confirm that that's not working for you and where exactly does it differ? Do you see the scoped registry (in `Project Settings` under `Package Manager`)?

Can you please add a link to the mentioned MRTK3 instructions that you followed?

I understand that this setup seems complicated for developers not that familiar with Unity, but let me comfort you by saying an easier method to install is in the making.

**Comment by laultman** [comment]
@atteneder Thanks for answering. I had to move on as there was no more time available. The team reprioritized the project and that part got bumped for now.
I jumped on Unity to do a very specific task. I needed to test a concept for a project. A code package had been developed several years back in Unity supporting HoloLens 2. I was trying to avoid having to convert code package to a different render engine. I was hoping we might use Unity for some of our small supporting efforts for individual customers where the projects are related but will not become part of our product codebase.
Unity still very difficult to manage. Although I was very pleased that the remote rendering has removed one of our largest issues with Unity - dev cycle time.
